src/main/java/org/breedinginsight/brapi/v1/model/Method.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.breedinginsight.brapi.v1.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder
+@NoArgsConstructor
+public class Method {
+    @JsonProperty("class")
+    private String propertyClass;
+    private String description;
+    private String formula;
+    private String methodDbId;
+    private String methodName;
+    private String name;
+    private OntologyReference ontologyRefernce;
+    private String reference;
+}
```

Comment:
Talked to Nick about these custom brapi v1.3 models in our bi-api repo. Conclusion was that another card can be created to explore getting the swaggerhub models to be jackson annotated instead of gson so that we can use them in bi-api instead of storing self-defined brapi models.

Reply:
Added card https://breedinginsight.atlassian.net/browse/BI-668?atlOrigin=eyJpIjoiY2YzMTYyYmMyNzZjNGViYWE5ZjdkNDljNGFmZGQ4MWQiLCJwIjoiaiJ9